src/App.jsx

```diff
@@ -1,18 +1,28 @@
-import Footer from "./components/Footer/Footer";
-import Navbar from "./components/Navbar/Navbar";
-import Login from "./pages/Authentication/Login/Login";
-import Logout from "./pages/Authentication/Logout/Logout";
-import Signup from "./pages/Authentication/Signup/Signup";
-import Cart from "./pages/Cart/Cart";
-import Home from "./pages/Home/Home";
-import ProductListing from "./pages/ProductListing/ProductListing";
-import Wishlist from "./pages/Wishlist/Wishlist";
+import { Route, Routes } from "react-router-dom";
+import { Navbar, Footer } from "./components/index";
+import {
+  Home,
+  Cart,
+  Login,
+  Signup,
+  Logout,
+  ProductListing,
+  Wishlist,
+} from "./pages/index";
```

Comment:
You don't need to write index over here, just wrting `./pages` would also work, it is the advantage of using index.js for exports, React automatically detects that.

Reply:
Thank you for explaining that. Made the changes!